Userland/Libraries/LibWeb/HTML/HTMLFormElement.idl

```diff
@@ -1,6 +1,12 @@
 #import <DOM/HTMLFormControlsCollection.idl>
 #import <HTML/HTMLElement.idl>
 
+// https://html.spec.whatwg.org/multipage/forms.html#attr-form-autocomplete
+enum Autocomplete {
+    "on",
+    "off"
+};
```

Comment:
The invalid value default and missing value default for this attribute should both be "on". 

You can do this with the `MissingValueDefault` and `InvalidValueDefault` attributes.

The following script should output "on", "on", "off" to the console:
```html

<!DOCTYPE html>
<form id="form1"></form>
<form id="form2" autocomplete="invalid"></form>
<form id="form3" autocomplete="off"></form>
<script>
    console.log(form1.autocomplete);
    console.log(form2.autocomplete);
    console.log(form3.autocomplete);
</script>
```

Reply:
Good call, you are right. I've pushed the fix